app/controllers/concerns/blacklight/catalog.rb

```diff
@@ -83,7 +83,12 @@ def facet
     @facet = blacklight_config.facet_fields[params[:id]]
     raise ActionController::RoutingError, 'Not Found' unless @facet
 
-    @response = search_service.facet_field_response(@facet.key)
+    query_fragment = params[:query_fragment] || ''
+    @response = if query_fragment.present?
```

Comment:
Do we need `query_fragment`?

```suggestion
    @response = if params[:query_fragment].present?
```